src/index.js

```diff
@@ -22,12 +24,15 @@ const LOG = message => {
 
 const hooks = {
     release: releaseHook,
-    github: githubHook
+    github: githubHook,
+    hipchat: hipchatHook,
+    mxapp: mxappHook
 };
+
 const handleWebhook = request => {
-    LOG("Received webhook!");
-    const trigger = nconf.get("dev")
-        ? request.body
+    LOG("Received webhook!");           //ENV VAR BOOLS OPERATE ON THE
+    const trigger = nconf.get("dev")    //PRESENCE OF ANY STRING
+        ? request.body                  //"TRUE" AND "FALSE" WILL BOTH BE TRUE.
```

Comment:
pls put this in a better formatted block comment rather than this

Reply:
?